test/async-hooks/testcfg.py

```diff
@@ -3,4 +3,4 @@
 import testpy
 
 def GetConfiguration(context, root):
-  return testpy.AsyncHooksTestConfiguration(context, root, 'async-hooks')
+  return testpy.ParallelTestConfiguration(context, root, 'async-hooks')
```

Comment:
Do we want to change this back so that if someone changes something in the way `parallel` tests are run, they aren't surprised that it also affects `async-hooks` tests?